package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "private": "true"
+}
```

Comment:
Yarn normally generates a `package.json` with name, version, license, etc. We don't need any of that. Just `private` will suffice.